src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -220,6 +220,10 @@ protected function getContainerBuilder()
 
     private function findProperServiceName(InputInterface $input, SymfonyStyle $io, ContainerBuilder $builder, string $name, bool $showHidden)
     {
+        if (0 === strpos($name, '\\')) {
```

Comment:
Could we use this simpler code?

```php
$name = ltrim($name, '\\');
```

Reply:
Why didn't I think of that myself. So much better! Will change.

Reply:
Done